src/badguy/walking_badguy.cpp

```diff
@@ -190,6 +190,18 @@ WalkingBadguy::collision_badguy(BadGuy& , const CollisionHit& hit)
   return CONTINUE;
 }
 
+void
+WalkingBadguy::collision_tile(uint32_t tile_attributes)
+{
+  if(tile_attributes & Tile::HURTS && max_drop_height > -1 && on_ground())
```

Comment:
Add space after if. Maybe add parantheses around first part for better readability.

Reply:
After what? I tried to mimic the other functions syntax

Reply:
Oh okay I get it